alacritty/src/config/window.rs

```diff
@@ -2,58 +2,60 @@ use std::os::raw::c_ulong;
 
 use serde::Deserialize;
 
-use crate::config::{
-    failure_default, from_string_or_deserialize, option_explicit_none, Delta, FromString,
+use alacritty_terminal::config::{
+    failure_default, from_string_or_deserialize, option_explicit_none, FromString,
 };
-use crate::index::{Column, Line};
+use alacritty_terminal::index::{Column, Line};
+
+use crate::config::ui_config::Delta;
 
 /// Default Alacritty name, used for window title and class.
 pub const DEFAULT_NAME: &str = "Alacritty";
 
 #[serde(default)]
 #[derive(Deserialize, Debug, Clone, PartialEq, Eq)]
 pub struct WindowConfig {
-    /// Initial dimensions
+    /// Initial dimensions.
     #[serde(deserialize_with = "failure_default")]
     pub dimensions: Dimensions,
 
-    /// Initial position
+    /// Initial position.
     #[serde(deserialize_with = "failure_default")]
     pub position: Option<Delta<i32>>,
 
-    /// Pixel padding
+    /// Pixel padding.
     #[serde(deserialize_with = "failure_default")]
     pub padding: Delta<u8>,
 
-    /// Draw the window with title bar / borders
+    /// Draw the window with title bar / borders.
     #[serde(deserialize_with = "failure_default")]
     pub decorations: Decorations,
 
-    /// Spread out additional padding evenly
+    /// Spread out additional padding evenly.
     #[serde(deserialize_with = "failure_default")]
     pub dynamic_padding: bool,
 
-    /// Startup mode
+    /// Startup mode.
     #[serde(deserialize_with = "failure_default")]
     startup_mode: StartupMode,
 
-    /// Window title
+    /// Window title.
     #[serde(default = "default_title")]
     pub title: String,
```

Comment:
I'm not sure where to put this thing. It makes sense as for a window title, but on the other hand terminal also has its own title and in some cases they could be different (imagine tmux like program with multiple panes). Each pane is a terminal and has it's own title, however the resulted title is a title of the currently active pane.